src/components/alert/Alert.js

```diff
@@ -6,7 +6,18 @@ import { Box } from '../box';
 import { Button } from '../button';
 import omit from 'lodash.omit';
 
-const Alert = ({ children, primaryAction, secondaryAction, title, ...otherProps }) => {
+const ALERT_TYPES = {
+  confirm: 'confirm',
+  destructive: 'destructive',
+  error: 'error',
+};
+
+const BUTTON_LEVELS = {
```

Comment:
It would actually make more sense to have this defined in the Button component and import it from there.

Reply:
I will change this, good point indeed!